src/components/Picker/AutoSuggestTest.tsx

```diff
@@ -0,0 +1,194 @@
+import * as Autosuggest from 'react-autosuggest';
+import * as React from 'react';
+import Card from './Card';
+import * as style from './Picker.scss';
+import Chip from '../Chip';
+
+// https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Regular_Expressions#Using_Special_Characters
+
+function escapeRegexCharacters(str:string) {
+  return str.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+}
+
+function renderSuggestion(suggestion:any, { isHighlighted, query }:any) {
+  const index = suggestion.name.toLowerCase().indexOf(query.toLowerCase());
+  const nameBefore = suggestion.name.slice(0, index);
+  const queryData = suggestion.name.slice(index, index + query.length);
+  const nameAfter = suggestion.name.slice(index + query.length);
+
+  if (isHighlighted) return <Card isHighlighted={true} image={suggestion.image} nameBefore={nameBefore} bold={queryData} nameAfter={nameAfter} email={suggestion.email}/>;
```

Comment:
You don't need to pass `isHighlighted={true}` by default if you just put `isHighlighted` is = true.
Also, the if else statement returns almost an identical `Card` component. To prevent this just return `Card` and have a const isHighlighted = true or false and pass it in `isHighlighted={isHighlighted}`

Reply:
makes so much sense